.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 statistics = True
 max-line-length = 80
-ignore = E501, B008, B011, W503
+ignore = E501, B008, B011, W503, B905
```

Comment:
B905 checks for [the `zip(..., strict=True)` parameter, which was added only in Python 3.10](https://peps.python.org/pep-0618/).